controllers/tc000201_auto_approve_with_disabled_drift_detection_test.go

```diff
@@ -0,0 +1,185 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	infrav1 "github.com/chanwit/tf-controller/api/v1alpha1"
+	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// +kubebuilder:docs-gen:collapse=Imports
+
+func Test_000201_auto_approve_with_disabled_drift_detection(t *testing.T) {
```

Comment:
This is the perfect numbering. Thank you so much for following the spec index convention!